yt/data_objects/selection_objects/cut_region.py

```diff
@@ -59,6 +59,7 @@ def __init__(
         validate_object(field_parameters, dict)
         validate_object(base_object, YTSelectionContainer)
         if base_object is not None:
+            # todo: handle this (where's the explicit deprecation ?)
```

Comment:
@matthewturk , this is a question for you. I believe you wrote the surrounding code.
I would like to know whether it's safe to remove this if-block or if I should add a call to `issue_deprecation_warning`.

Reply:
I believe I did write this.  But I think that we can safely remove this.

Reply:
thanks !